doc/builders/special/darwin-builder.section.md

```diff
@@ -45,10 +46,39 @@ nixos login: builder (automatic login)
 
 > Note: When you need to stop the VM, run `shutdown now` as the `builder` user.
 
+Because the builder is not running by default on port 22, you will also need to create a new file at `/etc/ssh/ssh_config.d/100-linux-builder.conf`:
+
+```
+Host linux-builder
```

Comment:
It might be worth generalizing the builder installation script to also create this file

Reply:
I added a module in `nix-darwin` for the macOS builder which also generates the file so it may not be necessary as the docs currently use `nix-darwin` to create the launchd daemons.

https://github.com/LnL7/nix-darwin/blob/master/modules/nix/linux-builder.nix